### PR TITLE
Bundle api

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## How to install a bundle from the UI?

The UI knows the list of apps which are within the bundle.
The ui can interate over all of them and run the install request for each individual app.
This was the ui can handle progress as well as individual error handling by itself.

Objections? @felixheidecke @hurradieweltgehtunter 

## Do we need an occ command to install a bundle?

@PVince81 @michaelstingl @butonic THX

### CODE REVIEW

**Comment by michaelstingl** [comment]
> Do we need an occ command to install a bundle?

@DeepDiver1975 occ command to install a bundle would be epic!

**Comment by PVince81** [comment]
Part of https://github.com/owncloud/market/issues/74.

@VicDeo please review

**Comment by felixheidecke** [comment]
Means I "just" fire all installations @ the same time and wait what happens?

**Comment by DeepDiver1975** [comment]
> Means I "just" fire all installations @ the same time and wait what happens? Why iterate?

name it as you like ;-) firing all at once is also a serialized process depending on the pov ;-)

File: lib/MarketService.php [hunk 1]
```diff
@@ -27,6 +27,7 @@
 use GuzzleHttp\Exception\ClientException;
 use OC\App\DependencyAnalyzer;
 use OC\App\Platform;
+use OCA\Enterprise_Key\EnterpriseKey;
```
Comment (VicDeo) [inline lib/MarketService.php]: @DeepDiver1975 a cross-app dependency?

**Pull request merged**